src/commands/db.rs

```diff
@@ -450,17 +482,14 @@ fn submits(conn_cfg: DbConnectionConfig<'_>, matches: &ArgMatches) -> Result<()>
 }
 
 /// Implementation of the "db jobs" subcommand
-fn jobs(conn_cfg: DbConnectionConfig<'_>, config: &Configuration, matches: &ArgMatches) -> Result<()> {
+fn jobs(
+    conn_cfg: DbConnectionConfig<'_>,
+    config: &Configuration,
+    matches: &ArgMatches,
+) -> Result<()> {
     let csv = matches.get_flag("csv");
     let hdrs = crate::commands::util::mk_header(vec![
-        "Submit",
-        "Job",
-        "Time",
-        "Host",
-        "Ok?",
-        "Package",
-        "Version",
-        "Distro",
+        "Submit", "Job", "Time", "Host", "Ok?", "Package", "Version", "Distro",
     ]);
```

Comment:
This'll produce another "large" diff if we extend the vector but I guess that's acceptable (at least it's consistent (based on the length)).